src/Converter/ShipmentLineItemsConverter.php

```diff
@@ -45,19 +52,45 @@ private function convertUnitRefundToLineItem(ShipmentRefund $shipmentRefund): Li
         ;
         Assert::notNull($shippingAdjustment);
 
+        /** @var ShipmentInterface $shipment */
```

Comment:
Not exactly related to PR, but would be cool to fix as well.

```diff
--- private function convertUnitRefundToLineItem(ShipmentRefund $shipmentRefund): LineItemInterface
+++ private function convertShipmentRefundToLineItem(ShipmentRefund $shipmentRefund): LineItemInterface
```
and this by plugin notation should be `array $shipments`, but it's interfaced, so not sure what to do with it.
>    public function convert(array $units): array

Reply:
Fixed in https://github.com/Sylius/RefundPlugin/pull/271